documentation/ipi-install/modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -0,0 +1,31 @@
+
+[id='ipi-install-firmware-requirements-for-installing-with-virtual-media_{context}']
+
+= Firmware requirements for installing with virtual media
+
+The installer for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with RedFish virtual media. The following table lists supported firmware for installer-provisioned {product-title} clusters deployed with RedFish virtual media.
+
+.Firmware Compatibility for RedFish Virtual Media
+[frame="topbot", options="header"]
+|====
+|Hardware| Model | Management | Minimum Firmware Version
+.2+| HP | 10th Generation | iLO5 | v1.9
```

Comment:
I would talk with Roger lopez to see the min version for HP (afaik there is no limitation, like minimum firmware version like Dell)

Reply:
@iurygregory , I left those based on the Google doc that John pointed me too which I thought you were the owner off. I thought maybe you knew something I didn't :) 

If that was not the case, I'm not sure what we want to put as min firmware version for HP as the versions I tested just worked out of the box.

Reply:
@iurygregory The document Roger is talking about is here: https://docs.google.com/document/d/1UvBdMg5VfNn76n2cMwQiMnShFVAyDtTFgfjVpD7H8bw/edit

Reply:
@rlopez133 @johnwilkins this doc was just an idea (the information in the doc was just an example of what we can have..) I think we should just document limations for the hardware we know (if HP has no limitations we just say something like "at the moment there is no limitation for HP hardware"?)

Reply:
'known limitation' ?

Reply:
@iranzo yeah, we would document the known limitations we know for virtual media for each hardware